File: src/tests/sort.zig
const std = @import("std");

pub fn main() void {
    var arr: [100]u32 = undefined;
    var i: u32 = 0;
    while (i < 100) : (i += 1) {
        arr[i] = 100 - i;
    }
    i = 0;
    while (i < 99) : (i += 1) {
        var j: u32 = 0;
        while (j < 99 - i) : (j += 1) {
            if (arr[j] > arr[j + 1]) {
                const tmp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = tmp;
            }
        }
    }
    std.debug.print("{}\n", .{arr[50]});
}

File: src/tests/sort.rs
fn main() {
    let mut arr: [u32; 100] = [0; 100];
    for i in 0..100 {
        arr[i] = (100 - i) as u32;
    }
    for i in 0..99 {
        for j in 0..(99 - i) {
            if arr[j] > arr[j + 1] {
                let tmp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = tmp;
            }
        }
    }
    println!("{}", arr[50]);
}
